q2_feature_classifier/custom.py

```diff
@@ -15,7 +15,11 @@
 
 class LowMemoryMultinomialNB(MultinomialNB):
     def __init__(self, alpha=1.0, fit_prior=True, class_prior=None,
-                 chunk_size=-1):
+                 chunk_size=0):
```

Comment:
As Jai has commented elsewhere, I think the solution here for the moment is better documentation.